### DISCUSSION TITLE
Nothing getting written to shims directory

### DISCUSSION THREAD

**Discussion by cking-spire** [pyenv/pyenv#2659] [Q&A]
I am struggling to get pyenv-win to work. I have installed pyenv and installed several python versions. The shims directory is being added to my PATH. However, nothing ever get written to the shims directory. Whenever I run python, powershell just finds the system python. Does anybody have any suggestions what to try to fix this?

Thanks, 
Chris

**Comment by cking-spire** [comment]
This seems like it might be a clue:
PS D:\code\OpenPype> pyenv rehash
C:\Users\cking\.pyenv\pyenv-win\libexec\pyenv.vbs(0, 1) Microsoft VBScript runtime error: This key is already associated with an element of this collection

**Reply by cking-spire** [reply]
I was able to fix my own problem. .py was listed twice in my PATHEXT environment variable.

**Comment by native-api** [comment]
[Pyenv-Win is a different product](https://github.com/pyenv-win/pyenv-win).
